dotCMS/src/main/java/com/dotcms/rest/api/v1/system/role/RoleLayoutForm.java

```diff
@@ -0,0 +1,52 @@
+package com.dotcms.rest.api.v1.system.role;
+
+import com.dotcms.rest.api.Validated;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import java.util.Set;
+
+@JsonDeserialize(builder = RoleLayoutForm.Builder.class)
+public class RoleLayoutForm extends Validated {
+
+    private final String roleId;
+    private final Set<String> layoutIds;
+
+    private RoleLayoutForm(final RoleLayoutForm.Builder builder) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [It is a good practice to call super() in a constructor](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5164094)